cmd/ocsp-updater/main.go

```diff
@@ -395,25 +406,51 @@ func (updater *OCSPUpdater) missingReceiptsTick(batchSize int) {
 			continue
 		}
 	}
-
+	return nil
 }
 
 type looper struct {
-	clk       clock.Clock
-	stats     statsd.Statter
-	batchSize int
-	tickDur   time.Duration
-	tickFunc  func(int)
-	name      string
+	clk                  clock.Clock
+	stats                statsd.Statter
+	batchSize            int
+	tickDur              time.Duration
+	tickFunc             func(int) error
+	name                 string
+	recentFailures       int64
```

Comment:
I think we do want the backoff to increase just once, right? E.g:

fail; wait 1 s
fail; wait 2 s
fail; wait 4 s
succeed; wait 0 s
succeed; wait 0 s
fail; wait 1 s
fail; wait 2 s

etc.